dockerfiles/Dockerfile.apisix-openresty.deb

```diff
@@ -0,0 +1,22 @@
+ARG IMAGE_BASE="ubuntu"
+ARG IMAGE_TAG="20.04"
+
+FROM ${IMAGE_BASE}:${IMAGE_TAG}
+
+COPY ./utils/build-common.sh /tmp/build-common.sh
+COPY build-apisix-openresty.sh /tmp/build-apisix-openresty.sh
+COPY ./utils/determine-dist.sh /tmp/determine-dist.sh
+
+WORKDIR /tmp
+
+ARG version
+ARG IMAGE_BASE
+ARG IMAGE_TAG
+
+ENV IMAGE_BASE=${IMAGE_BASE}
+ENV IMAGE_TAG=${IMAGE_TAG}
+
+RUN version=${version} \
```

Comment:
Ditto, it should be `VERSION=${VERSION}`. Thanks.

Reply:
Changed to use `ENV` to set environment variables.